custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -509,6 +509,11 @@ def run_fit(self):
         if not self.options.output:
             self.options.output = mkdtemp()
             remove_temp_output = self.options.output
+        else:
+            if self.options.output == self.options.code_dir:
+                raise DrumCommonException(
+                    "The code directory may not be used as the output directory as well."
```

Comment:
can delete "as well".  the sentence reads better without it

Reply:
add to the "Raises" in the doc string, please

Reply:
1. please extract to a private method with a descriptive name
2. this doesn't and shouldn't be an `else`.  always check